src/sagemaker_huggingface_inference_toolkit/diffusers_utils.py

```diff
@@ -29,24 +29,22 @@ def is_diffusers_available():
 if is_diffusers_available():
     import torch
 
-    from diffusers import AutoPipelineForText2Image, DPMSolverMultistepScheduler, StableDiffusionPipeline
+    from diffusers import DiffusionPipeline
 
 
-class SMAutoPipelineForText2Image:
+class DiffusionPipelineForText2Image:
```

Comment:
Any particular reason to change the class name? Looks like it is only used here in this file, so should be safe.

Reply:
I did so as it was related to the import of the class `AutoPipelineForText2Image` in diffusers not used anymore. Maybe it would make more sense to name it `SMDiffusionPipelineForText2Image`